### PR TITLE
[INNOVX-129] removed travis.yml file and added codeowners workflow and metadata file

### PR DESCRIPTION
**Pull request by kaustuk-samsung**

### CODE REVIEW

**Review by x0id — CHANGES_REQUESTED** [review]
We must not add any company-related stuff, like codeowners and metadata to the public repository.
This is a fork of a public repo and should be treated accordingly!

File: .github/CODEOWNERS
Comment (pdeschen) [inline .github/CODEOWNERS:1]: Unfortunately, if you plan on keeping up to date with upstream, I don't think it make sense to have `CODEOWNERS` and `.metadata.yml` as this is really specific to our world.

**Review by pdeschen — COMMENTED** [review]
See comment. Remove `CODEOWNERS` and `.metadata.yml` and we should be good to go.

File: .github/workflows/pr-checks.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
```
Comment (pdeschen) [inline .github/workflows/pr-checks.yml]:
```suggestion
      - uses: actions/checkout@v3
```

**Comment by kaustuk-samsung** [comment]
@x0id @pdeschen I have addressed PR comments and removed code owners and metadata.yml file. Please review it

File: .github/workflows/pr-checks.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+name: PR Checks
+on:
```
Comment (x0id) [inline .github/workflows/pr-checks.yml]: There is an extra space at the end of line 3.

Comment (kaustuk-samsung) [reply .github/workflows/pr-checks.yml]: Ohh my bad 🙈 . Let me remove it and raise it again

Comment (pdeschen) [reply .github/workflows/pr-checks.yml]: To lint yml and/or GHA workflow (we shouldn't have to do manual linting:-), consider using either one of:

* https://github.com/adgear/and-action#yaml
* https://github.com/adgear/and-action#github-actions-workflows-

**Review by pdeschen — APPROVED** [review]
👍

**Review by x0id — APPROVED** [review]

**Pull request merged**